recipe_scrapers/bakels.py

```diff
@@ -0,0 +1,51 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class Bakels(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "bakels.com.au"
+
+    def author(self):
+        return "Australian Bakels"
+
+    def title(self):
+        return self.soup.find("h1").get_text()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        div = self.soup.find("div", id="tab-ingredients_1")
+        if div:
+            ingredients = div.find_all(class_="text-xs-left")
+            array = []
+            flag = False
+            for ingredient in ingredients:
+                text = ingredient.get_text(strip=True)
+                if text == "Ingredient":
+                    flag = True
+                elif flag:
+                    array.append(text)
+            return array
+
+    def instructions(self):
+        div = self.soup.find("div", id="tab-method_1")
+        if div:
+            tag = div.find("p")
+            if tag:
+                instructions = tag.get_text(separator="\n", strip=True)
+                instructions = "\n".join(
+                    step.split(". ", 1)[-1] for step in instructions.split("\n")
+                )  # Removes the instruction number from each step
+                return instructions
+
+    def cuisine(self):
+        return self.schema.cuisine()
+
+    def description(self):
+        if self.soup.find("main", class_="content").find("p"):
+            return self.soup.find("main", class_="content").find("p").get_text()
+        return "None"
```

Comment:
@As a fallback, which seems to be reliable across recipes you could look for the content in the `meta` tag.

The below example is from https://www.bakels.com.au/recipes/hot-cross-buns-using-baktem-red-v/, which doesn't have a visible description, but still has the meta tag.
```html
<meta name="description" content="A recipe for soft and sweet hot cross buns, perfect for Easter celebrations.">
```

```suggestion
        description_meta = self.soup.find("meta", {"property": "og:title"})
        if description_meta:
            return description_meta.get("content")
        return None
```

Reply:
I did notice not every website had a description. This is a good way of doing it and it seems pretty consistent from what I am gathering from testing on my local machine.